javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+var $contactForm = document.querySelector('#contact-form');
+
+var x = {};
```

Comment:
We typically want to avoid using one letter variable names, variable names should be descriptive of the values they hold. For example here something like `formValues` would be better.